qiskit/ignis/experiments/calibration/cal_table.py

```diff
@@ -66,7 +66,7 @@ def get_generator_kwargs(
             parameters: Optional[Union[str, List[str]]] = None,
             use_complex_amplitude: bool = True,
             remove_bad_data: bool = True
-    ) -> Dict[str, Union[int, float, complex]]:
+    ) -> Dict[str, Union[int, float, complex, circuit.Parameter]]:
```

Comment:
```suggestion
    ) -> Dict[str, Union[int, float, complex, Parameter]]:
```
You will probably also need to change the import statement.